src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestHelpers/BaseRunner.cs

```diff
@@ -118,47 +177,56 @@ private async Task<(TestResult, string)> RunAsync2(TestCase testCase)
             {
                 try
                 {
```

Comment:
nit: nested try could just be 

try {
}
catch (SetupHandlerException e)
{}
catch (Exception e)
{}

Reply:
Good catch - I had removed some of the code that was preventing it from being flattened.